source/_integrations/media_player.markdown

```diff
@@ -51,7 +51,7 @@ Available services: `turn_on`, `turn_off`, `toggle`, `volume_up`, `volume_down`,
 | `media_content_id`     |       no | A media identifier. The format of this is integration dependent. For example, you can provide URLs to Sonos and Cast but only a playlist ID to iTunes.                   |
 | `media_content_type`   |       no | A media type. Must be one of `music`, `tvshow`, `video`, `episode`, `channel` or `playlist`. For example, to play music you would set `media_content_type` to `music`. |
 | `enqueue`              |      yes | How the new media should interact with the queue. Must be one of `add`, `next`, `play`, `replace`. |
-| `announce`             |      yes | Set to `true` to request the media player to temporarily stop playing music to announce this media and then resume.
+| `announce`             |      yes | Set to `true` to request the media player to temporarily stop playing media to announce this media and then resume. Most media players don't support `announce` and will not resume playing media once the announcement finishes.
```

Comment:
Dangerous to put it with this wording because it requires updating it later.

```suggestion
| `announce`             |      yes | Set to `true` to request the media player to temporarily stop playing media to announce this media and then resume. Requires the media player to support this feature.
```

Reply:
OK, rephrased to explain what happens when the media player doesn't support `announce` while not quantifying support among integrations 👍